### PR TITLE
v1.0.0

### PR DESCRIPTION
**Pull request by philipp-heyse**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,12 @@
 language: node_js
-node_js: 9
-
+node_js: 10
```
Comment (sofib) [inline .travis.yml]: What is the reason for node 10 - aren't our extensions running on node 8?

File: .travis.yml [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
 install:
-  - cd extension; npm install; cd ..
-
+- npm install -g standard
```
Comment (sofib) [inline .travis.yml]: standard is available as npm script as package.json brings it.

File: .travis.yml [hunk 3]
```diff
@@ -1,9 +1,12 @@
...
+- cd extension; npm install; cd ..
 script:
-  - cd extension; npm audit; cd ..
-  - cd extension; npm run coverage; cd ..
+- cd extension; npm audit; cd ..
+- cd extension; standard; cd ..
```
Comment (sofib) [inline .travis.yml]: There are messages standard is producing but this is not failing travis build and it should.

File: .travis.yml (hunk 1 above)
Comment (philipp-heyse) [reply .travis.yml]: but 9 was fine? i updated to 10, since it will also give me an up2date npm, so i don't have to update during install.
-> npm audit

Comment (sofib) [reply .travis.yml]: I didn't say it was fine and I didn't complain as we talked about doing travis separately, which you did

**Pull request merged**